src/FaceDetect.py

```diff
@@ -10,38 +10,67 @@
 import dlib
 import cv2
 import RosMsgUtil
+import pickle
 
+def StartDetection(FrameQueue,RectQueue,FacepointQueue,SpeakerQueue):
```

Comment:
for all functions add doxygen format comments for the API reference on readthedocs (will be generated automatically)

- function description
- function's inputs with explanations and types
- function outputs & types